apps/labs/components/Posts/PostListItem/PostListItem.tsx

```diff
@@ -50,7 +50,10 @@ export const PostListItem: FC<TPostListItem> = ({ post, variant }) => (
         </Link>
       </h3>
       <p className="text-[1.2rem] font-normal leading-[2.7rem] text-black text-sans">
-        By {post.meta.author.fullName} {post.meta.published}
+        By {post.meta.authors[0].fullName}
+        {post.meta.authors.length > 1 &&
+          `+ ${post.meta.authors.length - 1} more`}{' '}
+        {post.meta.published}
```

Comment:
Looking forward to the page design... let's separate the names from the date.

```suggestion
        {` – ${post.meta.published}`}
```

Reply:
@bskinn about the " + # more" I need some details on how is should go.
I think " + # more" is a link right? if so what's going to be displayed on click.

Reply:
Currently on the Labs blog, the author+date line for each blog post in the [/blog gallery](https://labs.quansight.org/blog) is not linked, not clickable.

Let's not change that behavior with this PR. However, I think your question raises a potential design issue that needs to be addressed, does the " + n more" create an expectation for many users that the line should be clickable? I don't want to annoy users by putting things on the page that makes them think they should be able to click it but then it doesn't do anything.

Let's have Irina look at them once we've shipped this and ask her to make a decision about whether the text should be styled differently or linked.

Reply:
#758

Reply:
We could also ask Irina to look at the preview build of this PR and then this PR would close #758